app/src/test/java/uk/ac/ebi/atlas/search/metadata/HighchartsSunburstAdapterTest.java

```diff
@@ -75,18 +76,17 @@ void setUp() {
                                 species1OrganismPart2ExperimentAccession)));
     }
 
-    @DisplayName("Cell type wheel should not have the same value for the ID and its parent")
     @Test
-    void cellTypeWheelSunburstResultObjectShouldNotContainIdAndParentAttributeValuesSame() {
-        var result = subject.getCellTypeWheelSunburst(metadataSearchTerm,cellTypeWheelServiceMock
+    void responseShouldContainIdAndParenAttributesAreNonEmptyValues() {
+        var result = subject.getCellTypeWheelSunburst(metadataSearchTerm, cellTypeWheelServiceMock
                 .search(metadataSearchTerm, species1.getName()));
-        assertThat(result).hasSize(2);
-        assertThat(result.asList().get(0).get("id")).isNotEqualTo(result.asList().get(0).get("parent"));
-        assertThat(result.asList().get(1).get("id")).isNotEqualTo(result.asList().get(1).get("parent"));
+        assertFalse(result.isEmpty());
+        assertNotEquals("", result.asList().get(1).get("id"));
+        assertNotEquals("", result.asList().get(1).get("parent"));
     }
 
     @Test
-    void ShouldThrowAssertionErrorIfWheelResultIdAndParentAttributeValuesAreSame() {
+    void failsIfIdAndParentFieldValuesAreSame() {
```

Comment:
Again, “fails” isn’t what you’re testing. Your test name describes the conditions and/or expectations of good behaviour. If it helps, @ke4 sent this a while ago to our Slack channel: https://osherove.com/blog/2005/4/3/naming-standards-for-unit-tests.html.